config/s3conn.py

```diff
@@ -1,6 +1,6 @@
 import boto3
 from botocore.client import ClientError
-import config
+import aws_config as config
```

Comment:
arguments to get buckets use `config.access_id` needed to `import aws_config` in order to get access to vars